frappe/desk/form/linked_with.py

```diff
@@ -13,7 +13,7 @@
 
 
 @frappe.whitelist()
-def get_submitted_linked_docs(doctype, name, docs=None, linked=None):
+def get_submitted_linked_docs(doctype, name, docs=None, linked=None, visited = []):
```

Comment:
in Python you cannot set a default value to an object in the declaration. It should be like:
``` 
def get_submitted_linked_docs(doctype, name, docs=None, linked=None, visited = None):
    if not visited: visited = []
```

Reply:
interesting. thanks
https://nikos7am.com/posts/mutable-default-arguments/

I will change this and push